src/components/autocomplete/js/autocompleteController.js

```diff
@@ -203,7 +207,8 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
       $mdUtil.enableScrolling();
     }
 
-    angular.element($window).off('resize', positionDropdown);
+    angular.element($window).off('resize', onWindowResize);
```

Comment:
Yeah I know what you mean, but what I said, having as an extra function is more clean (having it as an variable at the top is ugly).